web-wallet/src/routes/(app)/dashboard/receive/+page.svelte

```diff
@@ -0,0 +1,30 @@
+<svelte:options immutable={true} />
+
+<script>
+  import { onDestroy, onMount } from "svelte";
+  import { TransferContract } from "$lib/containers";
+  import { IconHeadingCard } from "$lib/containers/Cards";
+  import { contractDescriptors, updateOperation } from "$lib/contracts";
+  import { operationsStore } from "$lib/stores";
+  import { mdiArrowTopRight } from "@mdi/js";
+
+  onMount(() => {
+    updateOperation("receive");
+  })
+
+  onDestroy(() => {
+    updateOperation("receive");
+  });
+</script>
+
+<IconHeadingCard
+  gap="medium"
+  heading="Transfer"
+  iconPath={mdiArrowTopRight}
+  reverse
+>
+  <TransferContract
+    descriptor={contractDescriptors[1]}
+    on:operationChange={({ detail }) => updateOperation(detail)}
```

Comment:
Should we rethink the `TransferContract`?
It's a bit confusing having this page update the operation value, then leave the freedom to the inner component to do the same.

Reply:
On second thought: as you're separating things: can we get rid of that "operation store" entirely?

Reply:
Basically, yes.  But perhaps in it's own PR?